source/globals.d.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable @typescript-eslint/consistent-indexed-object-style */
 
 type AnyObject = Record<string, any>;
+// TODO: Fix this type, there's no such `delegate` global type
 type Deinit = delegate.Subscription | Observer | MutationObserver | ResizeObserver | IntersectionObserver | VoidFunction;
```

Comment:
`delegate.Subscription` and therefore `Deinit` are `any` due to #5422 @cheap-glitch